platforms/Windows/devtools-amd64.wxs

```diff
@@ -1,30 +1,33 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why drop the XML encoding marker(s)?  Does the new WiX toolset object?

Reply:
Unlikely, but the converter removed them. I don't mind either way, I don't think we'll run into a tool that misbehaves because they are missing.